### ISSUE TITLE
Create a New NFT - Image

### ISSUE THREAD

**Issue by jhobsz** [SecretChainGirl-Foundation/stashh-nft-marketplace#126]
Slide bar for thumbnail: default to no blur, include a message about the slide bar - Ex: "slide the bar to the right to blur the image"

**Comment by scarylooking** [comment]
I tend to agree with this. I don't think people notice the blur bar, and it leads to an entire marketplace where everything is blurred out.

People make purchasing decisions with their senses, so blurring by default will make it more difficult for someone to make a buying decision unless they explicitly know they that they want a specific NFT.

Setting the blur to "off" by default would solve this issue, while still allowing creators to easily turn it on if they want to.
